src/legacy/core_plugins/visualizations/public/np_ready/public/plugin.ts

```diff
@@ -32,6 +58,7 @@ export interface VisualizationsSetup {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface VisualizationsStart {
   types: TypesStart;
+  showNewVisModal: typeof showNewVisModal;
```

Comment:
Why is `showNewVisModal` exported on the contract, and not as a static function?
Is it because of `I18nContext`?

Reply:
and it needs access to the vis types registry